wai-logger/Network/Wai/Logger.hs

```diff
@@ -112,13 +114,13 @@ data ApacheLoggerActions = ApacheLoggerActions {
 ----------------------------------------------------------------
 
 -- | Creating 'ApacheLogger' according to 'LogType'.
-initLogger :: IPAddrSource -> LogType -> IO FormattedTime
+initLogger :: ToLogStr user => IPAddrSource -> LogType -> Maybe (Request -> Maybe user) -> IO FormattedTime
```

Comment:
I would like to maintain the backward compatibility for all exported stuff.
Could you define `initiLoggerUser` or something instead of changing the signature of `initLogger`?

Reply:
Ok, makes sense. Made that change.